app/views/jobs/show.html.erb

```diff
@@ -5,10 +5,22 @@
   </div>
 
   <div class="d-flex" >
-  <p><%= @job.id %></p>
-  <p><%= @job.description %></p>
+  <p> Job id number <%= @job.id %> : <%= @job.user.first_name %> needs help on [<%= @job.due_date %>]
+  <br>
+  Job description : <%= @job.description %></p>
   </div>
 
+<p> There are currently pending bookings for this job:</p>
+<% test_array = [] %>
```

Comment:
This kind of logic is probably better handled in the controller with instance variables